TestApps/FileSystemNamedCollection.swift

```diff
@@ -0,0 +1,139 @@
+//
+// Copyright 2023 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may obtain a copy
+// of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under
```

Comment:
This file is in incorrect location. Can you move it to `AEPServices/Sources/storage`?